lib/ex_unit/lib/ex_unit.ex

```diff
@@ -350,11 +350,24 @@ defmodule ExUnit do
   Runs the tests. It is invoked automatically
   if ExUnit is started via `start/1`.
 
+  It accepts an optional list of modules to rerun
+  the tests of specified modules.
+
   Returns a map containing the total number of tests, the number
   of failures, the number of excluded tests and the number of skipped tests.
   """
-  @spec run() :: suite_result()
-  def run do
+  @spec run(list(atom)) :: suite_result()
```

Comment:
Since the spec change, do we need to add the `since` attribute?

Reply:
Good question but I don't think so! We can mention this difference in the docs. :)